.github/workflows/test_and_deploy.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: ["3.8"]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
Does 3.12 require changes to the build process itself? (E.g., no distutils?)

Reply:
It does change a bit in the higher versions, for example pkg_resources is deprecated. I have a fix for that on the Gymnasium branch in the FB version which I'll bring in here shortly. That said, I left the range as 3.8 to 3.11 since that's what Gymnasium supports in it's current release.

Reply:
I removed pkg_resource this weekend.